src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -30,6 +30,8 @@ public abstract class MultithreadedAnalyzeCommandBase<TContext, TOptions> : Plug
         internal ConsoleLogger _consoleLogger;
         internal ConcurrentDictionary<string, IAnalysisLogger> _analysisLoggerCache;
 
+        internal static int? s_numberOfThreadsForTesting;
```

Comment:
>s_numberOfThreadsForTesting [](http://example.com/codeflow?start=29&length=27)

You should pick up this value from the context.

Reply:
removed.